### PR TITLE
Feature/users router

### PR DESCRIPTION
**Pull request by mmm1513**

### CODE REVIEW

File: app/backend/db/crud/users.py [hunk 1]
```diff
@@ -26,6 +27,43 @@ def get_user(
     return db.query(User).filter(User.id == user_id).first()
 
 
+def delete_user(
+    db: Session,
+    user_id: int
+):
+    user = get_user(db=db, user_id=user_id)
+    db.delete(user)
```
Comment (IamShobe) [inline app/backend/db/crud/users.py]: Commit not needed?

File: app/backend/server/api/v1/users_router.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+@router.get("/me", response_model=schemas.User)
+async def get_current_user(
+    current_user: schemas.User = Security(auth.get_current_user,
+                                          scopes=["personal"])
+):
+    return current_user
+
+
+@router.get("/subjects", response_model=List[schemas.User])
```
Comment (IamShobe) [inline app/backend/server/api/v1/users_router.py]: Doesn't even make sense..
It should be under /users/{I'd}/subjects

File: Pipfile [hunk 3]
```diff
@@ -7,6 +7,7 @@ verify_ssl = true
 pycodestyle = "*"
 ipdb = "*"
 autopep8 = "*"
+rope = "*"
```
Comment (IamShobe) [inline Pipfile]: Unused?

File: app/backend/server/api/v1/madors_router.py [hunk 4]
```diff
@@ -3,18 +3,22 @@
...
 from db import schemas
 from db.database import get_db
 
 router = APIRouter()
 
-
-@router.get("/me", response_model=schemas.User)
-async def get_current_user(
-    current_user: schemas.User = Security(get_current_user,
+@router.get("/hierarchy", response_model=List[schemas.Hierarchy])
```
Comment (IamShobe) [inline app/backend/server/api/v1/madors_router.py]: What about the mador router we discussed?

File: app/backend/db/crud/users.py (hunk 1 above)
Comment (mmm1513) [reply app/backend/db/crud/users.py]: done

File: app/backend/server/api/v1/users_router.py (hunk 2 above)
Comment (mmm1513) [reply app/backend/server/api/v1/users_router.py]: why should i be able to access anyone elses subjects?

File: Pipfile (hunk 3 above)
Comment (mmm1513) [reply Pipfile]: done

File: app/backend/server/api/v1/madors_router.py (hunk 4 above)
Comment (mmm1513) [reply app/backend/server/api/v1/madors_router.py]: ive just copied my code and will continue this in another branch

File: app/backend/server/api/v1/users_router.py (hunk 2 above)
Comment (mmm1513) [reply app/backend/server/api/v1/users_router.py]: ive changed it. w/e

**Comment by IamShobe** [comment]
please add tests

**Comment by IamShobe** [comment]
close because merged with date datas issue PR

**Pull request closed without merging**